src/borg/platform/linux.pyx

```diff
@@ -219,7 +222,7 @@ cdef _sync_file_range(fd, offset, length, flags):
         raise OSError(errno.errno, os.strerror(errno.errno))
     safe_fadvise(fd, offset, length, 'DONTNEED')
 
-cdef unsigned PAGE_MASK = resource.getpagesize() - 1
+cdef unsigned PAGE_MASK = sysconf(_SC_PAGESIZE) - 1
```

Comment:
Hm? You can always do that.

Maybe you're thinking of signed and unsigned comparisons?